api/app/Http/Controllers/JobFormController.php

```diff
@@ -0,0 +1,156 @@
+<?php
+
+namespace App\Http\Controllers;
+
+use App\JobForm;
+use Illuminate\Http\Request;
+use Illuminate\Support\Facades\DB;
+
+class JobFormController extends Controller
+{
+    /**
+     * Create a new controller instance.
+     *
+     * @return void
+     */
+    public function __construct()
+    {
+        $this->middleware('auth');
+    }
+
+    /**
+     * Display a listing of the resource.
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function index()
+    {
+        return view('job-form.index',[
+            'forms' => JobForm::with('user')->get(),
+        ]);
+    }
+
+    /**
+     * Show the form for creating a new resource.
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function create()
+    {
+        return view('job-form.create',[
+            'fields' => $this->getTableColumns('job_forms')->reject(function ($column) {
```

Comment:
Couldn't this be created in the constructor? Wouldn't need to call the function twice, less cluttery. The database structure shouldn't change mid-call.

Reply:
Great idea, actually, improves readability. However does not make any more difference.

Reply:
done